config/src/config/api_config.rs

```diff
@@ -47,6 +50,7 @@ pub const DEFAULT_MAX_SUBMIT_TRANSACTION_BATCH_SIZE: usize = 10;
 pub const DEFAULT_MAX_PAGE_SIZE: u16 = 100;
 pub const DEFAULT_MAX_ACCOUNT_RESOURCES_PAGE_SIZE: u16 = 9999;
 pub const DEFAULT_MAX_ACCOUNT_MODULES_PAGE_SIZE: u16 = 9999;
+pub const DEFAULT_MAX_VIEW_GAS: u64 = 9999;
```

Comment:
Will need some suggestion from @vgao1996 for what's the right gas limit to use here.

Reply:
I might be missing some contexts here, but conceptually I think view functions should have nothing to do with gas -- they are executed on fullnodes, and should not result in any APT being deducted as gas fee. 

However, I do realize that we could use gas as a way to limit the running time of a view transaction. You can check the current maximum in `aptos-gas` and use that as a reference.

Reply:
Perhaps a useful model here is that those functions are metered as usual, however, the meter is under control of the fullnode calling.

Reply:
That's exactly what's happening now. We enabled metering as usual, the gas limit is set by full node's local config.

Reply:
For now, put the current maximum in `aptos-gas` and we can adjust accordingly afterwards for public good & defaults.